{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":86663,"time":"2017-11-07T02:34:45.136Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":86663,"time":"2017-11-07T02:34:45.294Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":86663,"time":"2017-11-07T02:34:48.866Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":86663,"time":"2017-11-07T02:34:48.867Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID 437400998660806077.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":86663,"time":"2017-11-07T02:34:52.233Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":86663,"time":"2017-11-07T02:34:52.233Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":13705,"time":"2017-11-08T14:00:06.002Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":13705,"time":"2017-11-08T14:00:06.165Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":13705,"time":"2017-11-08T14:00:15.302Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":13705,"time":"2017-11-08T14:00:15.303Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ0Njg0NTkyLGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMTg5MTE2MTk5OF0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":13705,"time":"2017-11-08T14:00:16.737Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":13705,"time":"2017-11-08T14:00:16.738Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14277,"time":"2017-11-08T14:24:38.072Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14277,"time":"2017-11-08T14:24:38.223Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14277,"time":"2017-11-08T14:24:41.505Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14277,"time":"2017-11-08T14:24:41.506Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ0ODcyNzI3LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMTYxMDU3NTY0NF0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14277,"time":"2017-11-08T14:24:43.256Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14277,"time":"2017-11-08T14:24:43.256Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:38:14.706Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:38:14.856Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:38:17.315Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:38:17.316Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ0OTc2OTM3LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMjgzODAwMDcyMF0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:38:18.752Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:38:18.753Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:39:19.246Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:39:19.247Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:40:19.179Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14431,"time":"2017-11-08T14:40:19.180Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14678,"time":"2017-11-08T14:54:42.620Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14678,"time":"2017-11-08T14:54:42.785Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14678,"time":"2017-11-08T14:54:45.468Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14678,"time":"2017-11-08T14:54:45.469Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ1MDk5OTAxLGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMzM1NTIxNDczN10.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14678,"time":"2017-11-08T14:54:48.215Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":14678,"time":"2017-11-08T14:54:48.215Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:07:58.502Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:07:58.668Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:08:01.141Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:08:01.142Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ1NzA5MzM3LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMTcxMjg2NzcxMl0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:08:02.477Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:08:02.478Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:09:02.953Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:09:02.953Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:10:02.929Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:10:02.929Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:11:03.054Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:11:03.055Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:12:02.939Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:12:02.939Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:13:02.922Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:13:02.923Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:14:03.076Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:14:03.076Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:15:02.933Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:15:02.934Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:16:02.929Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:16:02.929Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:17:03.034Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:17:03.034Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:18:02.919Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:18:02.919Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:19:02.969Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:19:02.970Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:20:03.066Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:20:03.067Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:21:02.939Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:21:02.939Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:22:02.943Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:22:02.944Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:23:03.571Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:23:03.571Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:24:02.970Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:24:02.970Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:25:02.936Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:25:02.936Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:26:03.239Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:26:03.239Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:27:03.459Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:27:03.460Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:28:03.978Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:28:03.978Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:29:03.103Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:29:03.103Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:30:03.980Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15356,"time":"2017-11-08T16:30:03.981Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:31:56.388Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:31:56.541Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:32:03.238Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:32:03.240Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ1OTE3MTA4LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMzYxMDYzNjE5XQ.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:32:04.678Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:32:04.679Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:33:05.121Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:33:05.122Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:34:05.119Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:34:05.120Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:35:05.321Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:35:05.321Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:36:05.461Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:36:05.462Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:37:05.112Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:37:05.113Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:38:05.901Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:38:05.901Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:39:05.200Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:39:05.200Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:40:05.167Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:40:05.167Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:41:05.266Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":15931,"time":"2017-11-08T16:41:05.266Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:47:20.767Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:47:20.929Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:47:23.409Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:47:23.410Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ2MDMwNjk5LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sNTkzMzM3OTU1XQ.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:47:24.804Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:47:24.805Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:48:25.238Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16098,"time":"2017-11-08T16:48:25.239Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16853,"time":"2017-11-08T17:38:42.435Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16853,"time":"2017-11-08T17:38:42.585Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16853,"time":"2017-11-08T17:38:46.630Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16853,"time":"2017-11-08T17:38:46.631Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ2NDgyMjU4LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMjgxOTY2MjcxXQ.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16853,"time":"2017-11-08T17:38:48.463Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16853,"time":"2017-11-08T17:38:48.463Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:39:18.137Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:39:18.239Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:39:20.584Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:39:20.586Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6MjQ2NDg5NDU0LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sNzg4MDkwNDk0XQ.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:39:21.915Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:39:21.916Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:40:22.379Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":16870,"time":"2017-11-08T17:40:22.380Z","msg":"Error on submission to New Relic (data held for redelivery):","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":23112,"time":"2017-11-29T21:50:42.170Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":23112,"time":"2017-11-29T21:50:42.327Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":23112,"time":"2017-11-29T21:50:44.604Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":23112,"time":"2017-11-29T21:50:44.605Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6NDU2OTAyMDg4LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMTcyNTczNTY1MF0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":23112,"time":"2017-11-29T21:50:45.856Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":23112,"time":"2017-11-29T21:50:45.858Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":32896,"time":"2017-11-30T14:14:48.286Z","msg":"Using New Relic for Node.js. Agent version: 2.3.2; Node version: v6.10.1."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":32896,"time":"2017-11-30T14:14:48.840Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":32896,"time":"2017-11-30T14:15:03.440Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/1791929/applications/101827557","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":32896,"time":"2017-11-30T14:15:03.441Z","msg":"Connected to collector-237.newrelic.com:443 with agent run ID WzIse2E6NDYzODIzOTQ0LGI6MTAxODQwMzU2LGM6MTc5MTkyOSxkOiIyLjMuMiIsZToibm9kZWpzIixmOiJhbGV4YW5kZXJzLU1hY0Jvb2stUHJvLTIubG9jYWwiLGc6W3thOjEwMTgyNzU1NyxiOiJUcmltbWVyV29yayJ9XX0sMTUxNDk0ODc5Ml0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":32896,"time":"2017-11-30T14:15:07.428Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
{"v":0,"level":50,"name":"newrelic","hostname":"alexanders-MacBook-Pro-2.local","pid":32896,"time":"2017-11-30T14:15:07.429Z","msg":"New Relic for Node.js halted startup due to an error:","message":"Got HTTP 401 in response to metric_data.","stack":"Error: Got HTTP 401 in response to metric_data.\n    at StreamSink.parser [as callback] (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/collector/parse-response.js:99:19)\n    at StreamSink.end (/Users/asaldivar/freelance/trimmerwork/client/dist/node_modules/newrelic/lib/util/stream-sink.js:43:8)\n    at IncomingMessage.onend (_stream_readable.js:511:10)\n    at IncomingMessage.g (events.js:291:16)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)","statusCode":401}
